### DISCUSSION TITLE
Balancing of Book of Nine Swords and Path of War content in a WotR mythic setting

### DISCUSSION THREAD

**Discussion by ReggyDawn** [V0idhead/WOTRNineSwords#4] [Ideas]
Hi! Pathfinder veteran here. Have played it in all possible iterations. Gestalt, mythic, level 20, level 20+, even a combination of all of these at once, and then some additional 3pp subsystems on top. I wanted to give some commentary after having tested your mod in actual gameplay, and this commentary unsurprisingly aligns with my experience play tabletop.

The issue is that your mod is effective only in the context of using Time Stands Still, Mithral Tornado and the Diamond Mind equivalent of Mythic Vital Strike. All other strikes, surprisingly or not, are very much overshadowed by Mythic Vital Strike utterly and completely. You might argue that the feat requirements of mythic vital strike, as well as the fact that it doesn't synergize with Legend's high BAB or Demon's Superhaste aspect would be a point against mythic vital strike, but even then, only in the context of Time Stands Still.

Unfortunately much of 3.5 is quite outdated in a mythic pathfinder setting, and even the majority of PoW's strikes also tend to fall to the wayside in this manner, with the main benefit being boosts and counters that you might use along with your main form of attack, which is rarely strikes. 

For this reason, I would suggest you keep this in mind as you go forward with your mod. Either considering making strike maneuvers work with Mythic Vital Strike to provide a credible damage bonus, or making a Mythic Strike feat equivalent that adds some credible damage bonus and DC scaling to lower level single attack existing strikes so that they are kept relevant. 

Other than that, I love the idea. PoW / Bo9S always spiced up martial combat for me, and made it a reactive back and forth, rather than a passive stat check. If you manage to recreate that in Wrath along with counters, this will be my favorite mod of all time.

**Comment by V0idhead** [comment]
I really appreciate your input, especially since my own experience is limited to reading Bo9S and trying it out in WotR.

During my own playtesting I came to similar conclusions, but wanted start out with an implementation similar to the source material to keep it simple. However, I erred on the side of making strikes more powerfull when I had to make compromises due to technical issues.

My current plan is to make strikes with secondary effects (like stunning) more powerful by giving the player tools to raise the DC. I think that's a more interesting way to balance the class than simply raising the damage output beyond what a fighter could do.

Counters are on my todo-list, but the implementation will have to differ substantially from the rules as written because there is no way to request player input when the condition for a counter are met. The options that I came up with so far are either
 - Make counters a toggleable that always triggers when the condition is met (which has the potential to drain your available maneuvers extremely fast)
or
 - Turn counters into a boost, that has to be manually triggered during your turn.
Neither of these options seem particularly appealing, which is the reason I held off on implementing counters so far.

If you have any ideas for making counters work or other suggestions let me know, and I'll see what I can implement.

**Reply by ReggyDawn** [reply]
I've seen in a Swashbuckler mod Parry and Riposte, which is very close to a counter, be used as a toggle effect that triggers if the toggle is on and if the conditions are met. Perhaps adding a once per round limitation would be close enough for comfort. But yeah, there are no good ways to really implement it. It makes you wish that WotR had the same system as BG3 with their reactions pausing the game to offer you the option of triggering them, or not.

**Reply by V0idhead** [reply]
Swashbuckler is a good tip, thanks.
I already came up with an idea or two for possible implementations, I'll see what's doable after fixing a couple of bugs and finishing up a few new feats.

**Comment by V0idhead** [comment]
I just pushed https://github.com/V0idhead/WOTRNineSwords/releases/tag/1.3 which adds a handful of counters.
I think I came up with a good balance between power and ease of use:
Wall of Blades, Manticore Parry and Iron Heart Focus cost 2 maneuvers when their conditions are triggered (and they are toggled on), but they last for a round.
Lightning Recovery and Rapid Counter cost 1 maneuver whenever they trigger.

Feel free to let me know what you think about the balancing or encounter any problems.

**Reply by ReggyDawn** [reply]
I would suggest increasing maneuver counter cost for Wall of Blades and Manticore Parry to 3. 

Other than that, balance wise this has the effect of it being optimal to make sure you are offensively capable without relying on maneuvers so that you can use counters mostly for defensive purposes. Though a few offensive maneuvers are still often very worthwhile to use. Building an Improved Cleaving Finish Vital Strike or Full-Attack Warblade that uses maneuver solely for counters seems to be the most optimal way of doing it, especially with a few dips here and there to pad the numbers a bit more. But that's nothing particularly new, as it aligns with how ToB and PoW plays in tabletop mythic pathfinder, as I've said in the past.

Gameplay wise, it does lose a bit of its luster compared to tabletop, where you have to make the active decision to use it, rather than a passive toggle. It will also cut into the spectacle and fun of fighting enemies that use maneuvers as well, if you ever choose to go down that path. But that's just technical limitations, really. Other than that, it still introduces that desired dynamic martial fighting into the game, where you feel you have decisions to make, and that's a big win in my opinion.

I do so very wish the game had the ability to pause and give you options to trigger immediate actions, like BG3 does, though. So much more could be possible.

**Reply by V0idhead** [reply]
A cost of 3 seems a bit excessive to me.
But I just released 1.4.2 which adds a -1 penalty for each successful parry, that resets each round.
I might bump it to -2 if the parries are still too powerful.

**Reply by ReggyDawn** [reply]
> A cost of 3 seems a bit excessive to me. But I just released 1.4.2 which adds a -1 penalty for each successful parry, that resets each round. I might bump it to -2 if the parries are still too powerful.

A very elegant solution. I like it quite a bit more than mine!

**Comment by Xhanatos** [comment]
Could you explain somewhere on a lvl 1 ability what the max Maneuver counter is based on? From my testing right now it seems to be purely the intelligence modifier?
Besides that really love the mod, one of the most unique ones I have seen so far for the Pathfinder games.

**Reply by V0idhead** [reply]
The maximum number of maneuvers that can be performed per fight without recovering is based solely on level.
I'll add an explanation to the decription of the class with https://github.com/V0idhead/WOTRNineSwords/issues/14
